examples/ios/swift/ListSwiftUI/AppDelegate.swift

```diff
@@ -0,0 +1,29 @@
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
```

Comment:
We generally delete all of this auto-generated boilerplate that doesn't do anything in the samples to make it easier for people to find the actually relevant code.